node_common/utilities.js

```diff
@@ -132,7 +132,29 @@ export const getBucketAPIFromUserToken = async (token, user) => {
   let target;
   console.log(`[buckets] getOrCreate`);
   try {
-    target = await buckets.getOrCreate(BUCKET_NAME);
+
+    // Create a threads client
+    const client = new Client(buckets.context)
+    try {
+      // Get a default thread to store our buckets
+      const res = await client.getThread('buckets')
+      buckets.withThread(res.id.toString())
+    } catch (error) {
+      if (error.message !== 'Thread not found') {
+        throw new Error(error.message)
+      }
+      const newId = ThreadID.fromRandom()
+      await client.newDB(newId, 'buckets')
+      threadID = newId.toString()
+      buckets.withThread(threadID)
```

Comment:
I'd also like to get an idea of the actual thread id that is being used here. I saw some issues in the past with funny behavior around Buffer in thread-id. Can we log the id or otherwise debug what's actually making it to the bucket's context for that field? Even dumping the whole bucket context would be useful here.